### PR TITLE
systemd-analyze dump fixup

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/core/manager.c [hunk 1]
```diff
@@ -2094,13 +2094,15 @@ void manager_dump(Manager *m, FILE *f, const char *prefix) {
         assert(f);
 
         for (q = 0; q < _MANAGER_TIMESTAMP_MAX; q++) {
+                const dual_timestamp *t = m->timestamps + q;
                 char buf[FORMAT_TIMESTAMP_MAX];
```
Comment (yuwata) [inline src/core/manager.c]: Please update the size of the buffer.
```
char buf[FORMAT_TIMESPAN_MAX];
assert_cc(FORMAT_TIMESPAN_MAX >= FORMAT_TIMESTAMP_MAX);
```

Comment (poettering) [reply src/core/manager.c]: or maybe `char buf[CONST_MAX(FORMAT_TIMESPAN_MAX, FORMAT_TIMESTAMP_MAX)];`

Comment (keszybz) [reply src/core/manager.c]: Yep, good idea. Updated.

**Comment by yuwata** [comment]
LGTM.

**Pull request merged**